Added Custom Script doc for Publish (#501)
appcircleio/appcircle-docusaurus: docs/publish-integrations/common-publish-integrations/index.md

Appcircle's Common Integrations facilitate the distribution of your iOS and Android applications to major app stores with minimal effort.

## Custom Script

- Allows you to tailor the publish process to address your specific needs by executing custom commands.
- Automates repetitive tasks, reducing the amount of manual work required.

> - Allows you to create custom Publish flows and actions that are not available in the default steps.

Reviewer: ```suggestion
- Enables the creation of custom Publish flows and actions not available in the default steps.
```
